model_analyzer/model_manager.py

```diff
@@ -121,7 +129,7 @@ def run_models(self, models):
         self._server.update_config(params=server_config_copy.server_args())
 
         model_variant_name_manager_dict = self._state_manager.default_encode(
-            rcg._model_variant_name_manager)
+            rcg._model_variant_name_manager)  # type: ignore
```

Comment:
My boyscouting of type checking I exposed this error:
```../../model_analyzer/model_manager.py:132: error: "ConfigGeneratorInterface" has no attribute "_model_variant_name_manager"  [attr-defined]```

Don't know why it's showing up or how (or if we) need to fix this. I think it's safe to ignore this, as I don't see any real issue as MVNM is specified everywhere it needs to be.

Reply:
well accessing mvnm isn't part of the interface. If every ConfigGenerator is guaranteed to have one, it can/should be added to the interface. If not every one is guaranteed to have one, then this is unsafe code.

It looks like the mvnm is passed in to the object on line 108. Any reason we can't use `self._model_variant_name_manager` instead of `rcg._model_variant_name_manager`?

Reply:
I don't see why not. Changed.

Reply:
If this breaks anything, would your unit test catch it?  Can you run another CI before you merge?

Reply:
nevermind this is going into a side branch anyways

Reply:
I'm running a CI now. And, yes I believe unit tests would expose this, but I don't see an issue here as we are passing references since they are classes.